src/CMakeLists.txt

```diff
@@ -75,4 +75,6 @@ set(tfs_SRC
 	${CMAKE_CURRENT_LIST_DIR}/weapons.cpp
 	${CMAKE_CURRENT_LIST_DIR}/wildcardtree.cpp
 	PARENT_SCOPE)
+file(GLOB_RECURSE tfs_HDR *.h)
```

Comment:
If we use glob for headers, we should use for source files too, for the sake of consistency.